### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by suzi216**
Adding the code from trial

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  <body>
+<main id="main-doc">
+  <section class="main-section" id="Section_1">
+    <header>
+      <p>Section1</p>
+      <p>About <code>this</code>  section</p>
+      <li>Javascript</li>
+    </header>
+  </section>
```
Comment (suzi216) [inline index.html:17]: Please change the header tag to another semantic tag like div. Typically header is used for introductory content or a set of navigational links.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+<main id="main-doc">
+  <section class="main-section" id="Section_1">
+    <header>
+      <p>Section1</p>
+      <p>About <code>this</code>  section</p>
+      <li>Javascript</li>
```
Comment (suzi216) [inline index.html:15]: li tag usually comes with ul tag. Please change it accordingly to create an unordered list.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    </header>
+  </section>
+  <section class="main-section" id="Section_2">
+    <header>
+      <p>Section2</p>
+      <p>About <code>this</code>  section</p>
+      <li>Java</li>
+    </header>
+  </section>
```
Comment (suzi216) [inline index.html:24]: Please change all the section containers with the suggestions above.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      <li>R</li>
+    </header>
+    
+  </section>
+
+</main>
+<nav id="navbar">
+<header>
+  <h1>Diffrent programming language</h1>
+</header>
```
Comment (suzi216) [inline index.html:52]: Please place the header at the beginning of the page.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+<a href = "#" class = "nav-link">Section1</a>
+<a href = "#" class = "nav-link">Section2</a>
+<a href = "#" class = "nav-link">Section3</a>
+<a href = "#" class = "nav-link">Section4</a>
+<a href = "#" class = "nav-link">Section5</a>
```
Comment (suzi216) [inline index.html:57]: I would suggest adding the links in a container and make it easier to design them.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+#navbar {
+  position: fixed;
+  height: 100%;
+  top: 0;
+  left: 0;
+}
+@media screen and (max-height: 450px) {
+  #navbar  {padding-top: 15px;}
+  #navbar  a {font-size: 18px;}
+}
```
Comment (suzi216) [inline css/style.css:14]: Please add more css styles to arrange all the parts of the website in their natural order.

**Review by suzi216 — COMMENTED** [review]
- The correct flow is followed.
- The repo is documented in a professional way.

 **TO DO**
- Please add linters to you project
- Please check the inline comments to improve your code.

**Review by m05e5 — APPROVED** [review]
# Hi @Suzi216 , 

### Status: Approved ✔️
goodjob


:+1: . Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — COMMENTED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Suzi216 , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Good start with the review 👍 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**